### PR TITLE
PRODSEC-1255 Whitesource Transition to Snyk

### PR DESCRIPTION
**Pull request by sfc-gh-jfan**
Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am modifying authorization mechanisms
   - [ ] I am adding new credentials
   - [ ] I am modifying OCSP code
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

File: .github/workflows/snyk-pr.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: snyk-pr
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:5]:
```suggestion
      - main
```

File: .github/workflows/snyk-pr.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        path: whitesource-actions
+
+    - name: PR
+      uses: ./whitesource-actions/snyk-pr
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+      with:
+        jira_token: ${{ secrets.JIRA_TOKEN }}
+        gh_token: ${{ secrets.GITHUB_TOKEN }}
+        amend: false # true if you want the commit to be amended with the JIRA number
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:31]:
```suggestion
        amend: false
```

File: .github/workflows/snyk-pr.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        fetch-depth: 0
+
+    - name: checkout action
+      uses: actions/checkout@v2
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.WHITESOURCE_ACTION_TOKEN }}
+        path: whitesource-actions
+
+    - name: PR
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:24]: Please make this name more descriptive

File: .github/workflows/snyk-issue.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  whitesource:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout action
+      uses: actions/checkout@v2
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.WHITESOURCE_ACTION_TOKEN }}
+        path: whitesource-actions
+    - name: set-env
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-issue.yml:19]: Please be consistent with these names, some of them are all lower case and some of them aren't

File: .github/workflows/snyk-pr.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+name: snyk-pr
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  whitesource:
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:7]: Does this mean that we can get rid of the [other whitesource action](https://github.com/snowflakedb/snowflake-connector-python/blob/37443fb4cd0c548df0efe62199d2b52b324abe6e/.github/workflows/build_test.yml#L49)?

**Review by sfc-gh-mkeller — CHANGES_REQUESTED** [review]

**Comment by sfc-gh-jfan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/snyk-pr.yml (hunk 5 above)
Comment (sfc-gh-jfan) [reply .github/workflows/snyk-pr.yml:7]: yes, I have removed those files from the repo as well

**Review by sfc-gh-mkeller — APPROVED** [review]

**Pull request merged**